Loyc-Slim.sln

```diff
@@ -0,0 +1,395 @@
+
```

Comment:
I expected the tools you use - whatever they are - would support build configurations defined in the solution. Build configurations can exclude projects from building. But it wouldn't surprise me if, say, Xamarin Studio had flaky support for them.